### PR TITLE
Improve prompt.py style

### PR DESCRIPTION
**Pull request by mjakl**
**What current issue(s) does this address?, or what feature is it adding?**

The `prompt.py` responses were a bit inconsistent (casing, spacing etc.). This PR makes it a bit more consistent.

**How did you solve this problem?**

Adhere to PEP8 and the CoZ coding style used in other files.

**How did you make sure your solution works?**

No functional change.

**Did you add any tests?**

No.

**Are there any special changes in the code that we should be aware of?**

No.

### CODE REVIEW

File: prompt.py [hunk 1]
```diff
@@ -352,13 +349,13 @@ def do_export(self, arguments):
...
             if not self.Wallet:
-                return print("please open a wallet")
+                return print("Please open a wallet")
 
             address = get_arg(arguments, 1)
             if not address:
                 return print("Please specify an address")
 
-            passwd = prompt("[Wallet Password]> ", is_password=True)
+            passwd = prompt("[wallet password]> ", is_password=True)
```
Comment (metachris) [inline prompt.py]: Since most prompts now start capitalized, `wallet password` here should be `Wallet password`?

File: prompt.py [hunk 2]
```diff
@@ -791,50 +784,48 @@ def configure(self, args):
...
     def run(self):
-
         dbloop = task.LoopingCall(Blockchain.Default().PersistBlocks)
         dbloop.start(.1)
 
         Blockchain.Default().PersistBlocks()
 
         tokens = [(Token.Neo, 'NEO'), (Token.Default, ' cli. Type '),
-                  (Token.Command, "'help' "), (Token.Default, 'to get started')]
+                  (Token.Command, "\'help\' "), (Token.Default, "to get started")]
```
Comment (metachris) [inline prompt.py]: this has no unintended consequences?

File: prompt.py [hunk 3]
```diff
@@ -791,50 +784,48 @@ def configure(self, args):
...
 
         print_tokens(tokens, self.token_style)
-        print("\n")
+        print('\n')
```
Comment (metachris) [inline prompt.py]: why?

**Comment by metachris** [comment]
Nice cleanups 👍  

Some minor comments inline.

File: prompt.py (hunk 1 above)
Comment (mjakl) [reply prompt.py]: All prompts start with lower-case characters, only the responses are capitalised sentences.

Would you rather have the prompts capitalised as well? That's somewhat unusual.

File: prompt.py (hunk 2 above)
Comment (mjakl) [reply prompt.py]: Sorry, a leftover from the back/forth with the quotes.

File: prompt.py (hunk 3 above)
Comment (mjakl) [reply prompt.py]: It's the only `print("\n")` (1 out of 7), for the sake of consistency...

File: prompt.py (hunk 2 above)
Comment (mjakl) [reply prompt.py]: Since we mix the usage of `'` and `"` in the same array, I've used `'` consistently in the next commit.

**Comment by metachris** [comment]
Looks good to me, need to manually test it a bit more. @localhuman needs to sign off on this.

**Comment by localhuman** [comment]
Looks great, thanks for the contributions 😸

**Pull request merged**